src/doc/trpl/structs.md

```diff
@@ -87,3 +87,32 @@ fn main() {
     point.y = 6; // this causes an error
 }
 ```
+
+# Update syntax
+
+A `struct` can include `..` to indicate that you want to update just some
+values of a struct. For example:
```

Comment:
Umm, this does not indicate that. Instead this copies all other fields from another value of the same type.